src/routes/cars.routes.ts

```diff
@@ -9,8 +9,8 @@ const carsRouter = new Router();
 carsRouter.prefix('/cars');
 
 carsRouter.get("/", jwtAuth, permit(["user", "admin"]), validate(validationSchemas.getUserCarByIdSchema), carsController.getUserCarById);
-carsRouter.post("/add", jwtAuth, permit(["user", "admin"]), validate(validationSchemas.createCarSchema), carsController.createCar);
-carsRouter.put("/edit", jwtAuth, permit(["user", "admin"]), validate(validationSchemas.editCarSchema), carsController.editCar);
+carsRouter.post("/", jwtAuth, permit(["user", "admin"]), validate(validationSchemas.createCarSchema), carsController.createCar);
+carsRouter.put("/", jwtAuth, permit(["user", "admin"]), validate(validationSchemas.editCarSchema), carsController.editCar);
```

Comment:
Consider using `/:id` to make it clear that the endpoint updates a specific car by its ID, similar to the `DELETE` endpoint.

`carsRouter.put("/:id", jwtAuth, permit([Roles.User, Roles.Admin]), validate(validationSchemas.editCarSchema), carsController.editCar);`

Define the Roles Enum
```
enum Roles {
    User = "user",
    Admin = "admin"
}
```